app/pom.xml

```diff
@@ -89,7 +89,7 @@
     <dependency>
       <groupId>com.adobe.aem.dot</groupId>
       <artifactId>dispatcher-optimizer-core</artifactId>
-      <version>1.0.15-SNAPSHOT</version>
+      <version>1.0.17-SNAPSHOT</version>
```

Comment:
When we release 'core', I think we need to updated these too.  I couldn't build '15-SNAPSHOT'.